test/e2e/play_kube_test.go

```diff
@@ -3820,6 +3826,30 @@ invalid kube kind
 		Expect(checkls.OutputToStringArray()).To(BeEmpty())
 	})
 
+	It("podman play kube teardown with secret", func() {
+		err := writeYaml(secretYaml, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube).Should(Exit(0))
```

Comment:
I think we should check that the output of `play kube` also lists secrets.

Reply:
I had included that in the existing kube play test because we're specifically testing teardown here. Here-  https://github.com/containers/podman/pull/17526/files#diff-cef3dd63f572aaef036dce65d94140edbfa8c350c2fc836e864a6b1ef653db87R1649-R1653 Does this help?

Reply:
yes, thanks!